ci-operator/config/stolostron/multicluster-controlplane/stolostron-multicluster-controlplane-main.yaml

```diff
@@ -0,0 +1,96 @@
+base_images:
+  base:
+    name: ubi-minimal
+    namespace: ocp
+    tag: "8"
+  stolostron_builder_go1.19-linux:
+    name: builder
+    namespace: stolostron
+    tag: go1.19-linux
+build_root:
+  image_stream_tag:
+    name: builder
+    namespace: stolostron
+    tag: go1.19-linux
+images:
+- dockerfile_path: Dockerfile
+  from: base
+  inputs:
+    stolostron_builder_go1.19-linux:
+      as:
+      - registry.ci.openshift.org/stolostron/builder:go1.19-linux
+  to: multicluster-controlplane
+promotion:
+  disabled: true
+  name: "2.7"
+  namespace: stolostron
+releases:
+  initial:
+    integration:
+      name: "4.8"
+      namespace: ocp
+  latest:
+    integration:
+      include_built_images: true
+      name: "4.8"
+      namespace: ocp
+resources:
```

Comment:
Do you have any metrics that backs up the `memory` values?

Reply:
I refer to global hub for the `request memory` here, and I just set the `request memory` to `200Mi`, which should work.

Reply:
thanks